modules/ROOT/pages/questions/question-12.adoc

```diff
@@ -0,0 +1,16 @@
+12. The `media` plugin’s `media_scripts` option, which was deprecated
```

Comment:
@lnewson @metricjs

Reply:
> a. Does anyone know what the setting did?

Based on the code it looks like it was meant to be a way to provide an allow list of `<script>`s that could be used to embed media. However, somewhere along the lines that seems to have broke and no longer works. That's only a guess though, as it was created before I started at Tiny and there's little documentation and no tests for it. @metricjs might know more...

> b. Is it safe to just document the option’s removal, leaving
    discovery of whatever (currently unknown) functionality is lost to
    the end-user(s) who was, previously, making use of said unknown
    capability?

This seems fine to me, but also not really my question to answer. @EkimChau is probably the best one to answer that.

Reply:
a. No

b. I think yes, as if someone does contact us about it, at least then we can know what it actually did. Better than us guessing, I think.